clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/model/OpenstackInstance.groovy

```diff
@@ -16,29 +16,30 @@
 
 package com.netflix.spinnaker.clouddriver.openstack.model
 
+import com.fasterxml.jackson.annotation.JsonIgnore
 import com.netflix.spinnaker.clouddriver.model.HealthState
 import com.netflix.spinnaker.clouddriver.model.Instance
-import com.netflix.spinnaker.clouddriver.openstack.OpenstackCloudProvider
 import groovy.transform.EqualsAndHashCode
 
 import static org.openstack4j.model.compute.Server.Status
 
 @EqualsAndHashCode
 class OpenstackInstance implements Instance, Serializable {
-  final String providerType = OpenstackCloudProvider.ID
   String name
   String instanceId
-  long launchTime
+  @JsonIgnore
```

Comment:
Added JsonIgnore to launched at because the instance interface already has a launchTime method that returns the launchTime.  In other words, it was a duplicate value.